web/webkit/src/main/scala/net/liftweb/http/LiftRules.scala

```diff
@@ -2295,3 +2301,80 @@ trait FormVendor {
   private object requestForms extends SessionVar[Map[String, List[FormBuilderLocator[_]]]](Map())
 }
 
+abstract class LiftRulesSettingException(settingName: String, stackTrace: LiftRules.StackTrace, message: String) extends Exception(message) {
```

Comment:
This, and its attendant subclasses, should probably live in its own file. I know we've embraced putting 18000 classes in each file in Lift, but IMO we should move away from that :)